README.md

```diff
@@ -6,32 +6,19 @@
 [![Maven Central](https://img.shields.io/maven-central/v/com.auth0.android/auth0.svg?style=flat-square)](https://search.maven.org/artifact/com.auth0.android/auth0)
 [![javadoc](https://javadoc.io/badge2/com.auth0.android/auth0/javadoc.svg)](https://javadoc.io/doc/com.auth0.android/auth0)
 
```

Comment:
Any reason the other badges from the figma design (Circle status, license, coverage, etc.) are not added here?

Reply:
Excepts for downloads which is not available through maven, I guess we have covered everything @jimmyjames. And we have added `javadoc` badge to the mix

Design - 
![Screenshot 2022-10-19 at 16 48 25](https://user-images.githubusercontent.com/15910425/196725155-098ac7e1-9924-4027-a85e-384aa2e15332.png)

README preview -
<img width="529" alt="Screenshot 2022-10-19 at 16 48 17" src="https://user-images.githubusercontent.com/15910425/196725350-7bf8d981-e2aa-413b-8c61-f251726d01ad.png">